kong/api/routes/upstreams.lua

```diff
@@ -40,7 +42,48 @@ return {
     end,
 
     GET = function(self, dao_factory)
-      crud.paginated_set(self, dao_factory.targets)
+      -- by default, just get a paginated list
```

Comment:
we're adding code to determine the active ones (similar as when posting), can't this be refactored in a function, that can be called from both methods to prevent code duplication?